src/tailscale_svc_lb_controller/helpers.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import kubernetes
+
+import config
+
+
+def get_common_labels(service, namespace):
+    """
+    Get the labels common to all resources managed by this operator.
+    """
+
+    return {
+        "app.kubernetes.io/name": "tailscale-svc-lb",
+        "app.kubernetes.io/managed-by": "tailscale-svc-lb-controller",
+        config.SERVICE_NAME_LABEL: service,
+        config.SERVICE_NAMESPACE_LABEL: namespace
+    }
+
+
+def update_service_status(namespace, service, ip):
+    """
+    Update the status of the service to reflect the service Tailscale IP.
+    """
+
+    try:
+        # Get the service
+        k8s = kubernetes.client.CoreV1Api()
+        service_object = k8s.read_namespaced_service(name=service, namespace=namespace)
+
+        # Update the status
+        service_object.status.load_balancer.ingress = [
+            kubernetes.client.V1LoadBalancerIngress(ip=ip)
+        ]
+
+        # Patch the service with the new status
+        k8s.patch_namespaced_service_status(name=service, namespace=namespace, body=service_object)
+    except Exception as e:
+        raise e
```

Comment:
Since exception is not being acted up on, the whole `try...except` block can be removed:
```suggestion
    # Get the service
    k8s = kubernetes.client.CoreV1Api()
    service_object = k8s.read_namespaced_service(name=service, namespace=namespace)

    # Update the status
    service_object.status.load_balancer.ingress = [
        kubernetes.client.V1LoadBalancerIngress(ip=ip)
    ]

    # Patch the service with the new status
    k8s.patch_namespaced_service_status(name=service, namespace=namespace, body=service_object)
```